logger/options.go

```diff
@@ -37,6 +37,7 @@ type Options struct {
 	MaxBufferLen  int
 	Meta          string
 	Tags          string
+	Timestamp     int64
```

Comment:
in the `Options` this should probably be a `time.Time` instead of `int64` - we can do the conversion when added to the `Line`